tests/unit/cli/test_sanitizer.py

```
@@ -0,0 +1,42 @@
+    data = {
+        "message": "Token qdrant-secret should hide",
+        "tuple": ("sk-live", {"password": "hunter2"}),
+    }
+    scrubbed = sanitizer.scrub_object(data)
+    assert "qdrant-secret" not in scrubbed["message"]
+    assert scrubbed["tuple"][0] == "***"
+    assert scrubbed["tuple"][1]["password"] == "***"
+
+
+def test_scrub_object_sanitizes_tuples_in_lists():
+    payload = {"items": [("password", "sk-secret"), ("note", "contains sk-secret")]}  # noqa: S105 - test fixture
```

Ruff (0.13.1)

Unused `noqa` directive (unused: `S105`)

Remove unused `noqa` directive

(RUF100)